nvflare/lighter/poc.py

```diff
@@ -34,38 +32,72 @@ def clone_client(num_clients: int):
     shutil.rmtree(src_folder)
 
 
-def main():
-    parser = argparse.ArgumentParser()
-    parser.add_argument("-n", "--num_clients", type=int, default=1, help="number of client folders to create")
+def unpack_poc(src_poc_folder, dest_poc_folder):
+    file_dir_path = pathlib.Path(__file__).parent.absolute()
+    poc_zip_path = file_dir_path.parent / "poc.zip"
+    try:
+        shutil.unpack_archive(poc_zip_path)
+    except shutil.ReadError:
+        copy_from_src(src_poc_folder, dest_poc_folder)
 
-    args = parser.parse_args()
 
+def copy_from_src(src_poc_folder, dest_poc_folder):
+    try:
+        shutil.copytree(src_poc_folder, dest_poc_folder)
+    except BaseException as e:
+        print(f"Unable to copy poc folder from {src_poc_folder}, Exit. {e} ")
+        exit(1)
```

Comment:
From the function name, I think "unpack_poc" should NOT call "copy_from_src".

I think these two functions can be merged inside "clone_poc_folder".